src/phylum/ci/cli.py

```diff
@@ -105,6 +105,12 @@ def get_args(args: Optional[Sequence[str]] = None) -> Tuple[argparse.Namespace,
         action="store_true",
         help="Specify this flag to consider all dependencies in analysis results instead of just the newly added ones.",
     )
+    analysis_group.add_argument(
+        "-fa",
```

Comment:
Unfortunately a bit late now, but I feel like this is not a good idea. Usually on conventional unix CLIs `-fa` would imply `-f -a`, thus it's not really appropriate to have two-letter shorthands?

A shorthand shouldn't be necessary, but if it is used it should probably be a single letter. Using lower/uppercase for different meanings is however possible.

Reply:
#56 has been created to track this issue.